*This section briefly describes or lists the key points in each paragraph.*

#### p.2

* The functions and types of centromere
    - Mediates binding of the kinetochore and microtubules
    - Kinetochore assembles on histone H3 variant (CENP-A in this study)
    - Regional and point centromeres

* Fungal regional centromeres
    - The pericentric regions of centromeres in *Candida albicans*
        - Enriched: euchromatic histone marks H3K9Ac and H4K16Ac
        - Hypomethylated: H3K4, a feature of heterochromatic chromatin
        - Histone H3 is depleted at the core centromeres of *C. albicans* and H3K9 methylation is absent in the *C. albicans genome.
        - The mixture of euchromatin and heterochromatin results in reduced expression of genes located in proximity to the centromeres.
    - Native centromeres of *Schizosaccharomyces pombe*
        - CENP-A-enriched central core
        - Flanked by heterochromatic inner and outer repeats
    - The pericentric regions of centromeres in *Schizosaccharomyces pombe*
        - Enriched: H3K9me2/3, which is RNAi-dependent and essential for CENP-A localization to the central core.
    - The centromeres of basidiomycete *Cryptococcus neoformans* 
        - Enriched: full length and truncated transposable elements (these elements are silenced via RNAi)
        - Also enriched: heterochromatic marks H3K9me2 and 5mC DNA methylation.
        - However, heterochromatin formation in *C. neoformans* is not mediated by small RNAs (unlike *S. pombe*).

#### p.3

**About *C. deuterogattii***

* A sister species of *C. neoformans* is *C. deuterogattii* and this is the only member of the *Cryptococcus* pathogenic species complex that is RNAi-deficient.

* The loss of RNAi has been correlated with the presence of only truncated transposable elements within centromeres whose length has been significantly reduced as compared to RNAi-proficient *Cryptococcus* species.

* But phylogenetic analysis showed that the transposable elements in *C. deuterogattii* are still similar to those of other members of the species complex. The majority of these truncated centromeric transposable elements are Ty3-Gypsy family retroelements (Tcn1-Tcn5) with one Ty1-Copia element (Tcn6) among them.

**About neocentromere formation**

* Deletion of centromeres frequently results in neocentromere formation in genomic hotspots, and these are often formed in the vicinity of the native centromere.

* This phenomenon might be attributed to the presence of a "CENP-A clound".

* CENP-A cloud consists of free, non-incorportated CENP-A molecules that are present in close proximity to the centromeres at a concentration that decreases with increasing distance from the centromeres.

* Unlike the native centromeres in *C. albicans*, neocentromeres are not associated with flanking repeats and genes spanned by CENP-A are silenced.

* The lengths of centromeres varied from 2 to 7 kb, suggesting that neocentromeres in *C. albicans* can be flexible, unlike its native centromeres.

* Epigenetic properties of neocentromeres are not yet defined in *C. albicans*.

* Similar to *C. albicans*, the majority of chicken DT40 neocentromeres form in close proximity to the native centromere.

* Some findings suggest **heterochromatin is not essential for centromere function in DT40 cells and that heterochromatin may serve to repress only repetitive regions**.

* Neocentromeres in *S. pombe* form adjacent to heterochromatic regions, close to the telomeres, where pre-existing repeats mimic the higher-order repeat architecture of native centromeres.

* *S. pombe* neocentromeres can span genes that are silenced due to CENP-A binding (like *C. albicans*).

* Deletion of native centromeres in wild-type *S. pombe* leads to either neocentromere formation or chromosome fusion, while centromere deletion in H3K9me2-deficient cells results in only chromosome fusion, suggesting that **heterochromatin is essential for neocentromere formation and function in *S. pombe*.**

#### p.4

**Previous studies**

* The authors previously showed that deletion of a native centromere in *C. deuterogattii* leads to neocentromere formation ([ref](https://elifesciences.org/articles/56026){:target="_blank"}).

* Native centromeres span truncated transposable elements and are located in ORF-free regions. In contrast, neocentromeres of both chromosomes 9 and 10 span genes that continue to be expressed at wild-type levels.

* The neocentromeres are shorter in length and lack transposable elemtns.

* The wild-type expression of neocentromeric genes suggests neocentromeres may not be heterochromatic in *C. deuterogattii*.

* Neocentromeres of chromosome 10 were unstable and often led to chromosome-chromosome fusions between sub-telomeric regions. After fusion, the neocentromeres were inactivated and only the native centromere of the fusion partner chromosome served as the centromere of the fused chromosome.

**In this study**

1. The authors showed that deletion of the native centromere, derived from the fusion partner, in fused chromosomes results in neocentromere formation in close proximity to the active centromere, not at the previous neocentromere location. (Continuing from the last point of their previous study.)

2. In contrast to native centromeres, neoventromeres are not enriched for the heterochromatin mark H3K9me2 or 5mC DNA methylation and maintain their H3K4me2 status.

3. Integration of a selectable marker gene into a native centromere resulted in a loss of CENP-A binding at the native centromere and the formation of a neocentromere adjacent to the site of the original native centromere.

4. Overall, the results demonstrate that *C. deuterogattii* neocentromeres lack epigenetic marks that are present at native centromeres and do not retain any epigenetic memory following in activation.

*Introduction from the paper [Centromere deletion in Cryptococcus deuterogattii leads to neocentromere formation and chromosome fusions](https://elifesciences.org/articles/56026){:target="_blank"}*

1. Recently, the centromeres of the human pathogenic fungus *Cryptococcus deuterogattii* were characterized and compared to those of the closely related species *Cryptococcus neoformans* (centromeres ranging from 27 to 64 kb), revealing dramatically smaller centromeres in *C. deuterogattii* (ranging from 8.7 to 21 kb).
2. *C. deuterogattii* is responsible for an ongoing cryptococcosis outbreak in the Pacific Northwest regions of Canada and the United States. In contrast to the sister species *C. neoformans*, *C. deuterogattii* commonly infects immunocompetent patients.
3. *C. deuterogattii* is a haploid basidiomycetous fungus with 14 chromosomes. The dramatic reduction in centromere size in *C. deuterogattii* may be attributable to loss of the RNAi pathway. 

4. The centromeres of *C. deuterogattii* consist of **truncated transposable elements, and active transposable elements are missing throughout the genome**. This is in stark contrast to *C. neoformans*, which has active transposable elements in centromeric regions.

5. Neocentromeres are frequently formed near genomic repeats, yet *C. deuterogattii* lacks active transposons that might seed neocentromere formation. Thus, *C. deuterogattii* is a unique organism in which to study centromere structure and function.

