## Clr4-Mediated Histone H3 Lysine 9 Methylation Is an Essential Regulatory Mechanism in Mucorales

The Mucorales express 
a diverse repertoire of RNAi components that are involved in
distinct repressive, regulatory pathways relying on sRNA-directed 
degradation of complementary mRNAs that results in PTGS.

These sRNAs also serve as initiators of chromatin 
modifications leading to transcriptional gene silencing (TGS) in 
other eukaryotic organisms.

**While many of these modifications have been characterized in the Dikarya, their conservation and role in early-diverging fungi was unexplored. Therefore, the presence of chromatin modifications was first surveyed in early-diverging 
fungal genomes, focusing on the Mucorales** ([phylogeny](./images/2_intro_early_fungi.jpeg){:target="_blank"}). As in most eukaryotes, Mucoralean histone proteins H3 and H4 are highly conserved.  The M. lusitanicus genome encodes four histone H3 and 
three histone H4 copies (SI Appendix, Fig. S1 A and B), and the 
KMT-lysine substrates and neighboring regions are also conserved.  Thus, these proteins could harbor chromatin modifications 
catalyzed by KMT enzymes or other histone lysine modifiers.

### Methods

#### Results 1-6
To identify RNAi components, and KMT and DNMT homologs 
in the Mucorales, **86 fungal proteomes were analyzed including 
at least one representative of most fungal phyla, focusing on the  Mucoromycota phylum and major fungal pathogens**. 
Out of these proteomes, 43 were selected as representatives to 
capture the presence or absence of different epigenetic modifying enzymes. ([Fig 1](./images/2_F1.jpg){:target="_blank"})

*TODO: add method details*

#### Results 7-
To gain a better understanding of Clr4 function in Mucorales, 
we attempted to delete clr4 through **CRISPR-Cas9** and **homologous recombination** in *M. lusitanicus*.

Mucorales have nonseptate hyphae and multinucleated asexual spores. As such, genetic transformation often results in initially heterokaryotic mutants, producing spores that harbor both wild-type (WT) and mutant nuclei. Mutant allele segregation by sexual reproduction is not possible because *M. lusitanicus* is not able to undergo a complete sexual cycle under laboratory conditions. To render these mutants 
homokaryotic, i.e., harboring mutant nuclei exclusively, **transformants must be passaged through several vegetative growth cycles in a medium that positively selects the mutant allele**.



### Results 

1.  We identified homologs of the three major RNAi 
enzymes—RdRP, Dicer, and Ago—across all fungal phyla and 
particularly, in every Mucoromycota species, though not in some 
isolated Dikarya species or clades, e.g., *Saccharomyces cerevisiae*, *Candida auris*, *Ustilago maydis*, and *Malassezia pachydermatis*.

2. In *S. pombe*, CTGS leading to constitutive heterochromatin initiates in regions producing complementary sRNA that are loaded into the RITS complex by the Argonaute siRNA chaperones (ARC), Arb1 and Arb2. These two proteins are mostly conserved in the Mucorales, ascomycetes that have retained RNAi, and some early diverging fungi.

3. However, RITS-exclusive components—Chp1 and Tas3—are found only in *S. pombe*, indicating that any possible interplay among RNAi and heterochromatin formation might require novel components.

4. KMT homologs and accessory proteins in the methyltransferase complexes were identified. Clr4/KMT1, KMT6, and KMT5 catalyze methylation of H3K9, H3K27, and H4K20, respectively, resulting in transcriptional repression ([Fig S1](./images/2_S1.png){:target="_blank"}). Clr4 and associated proteins, Rik1 and the ubiquitin ligase Cul4, are highly conserved in fungi, particularly in the Mucorales. ([Fig 1](./images/2_F1.jpg){:target="_blank"})

5.  In *S. pombe*, the Clr4 complex (CLRC) has 
been shown to interact with Ago1 through Stc1, which connects 
both pathways to establish H3K9me via RNAi ([ref](https://link.springer.com/article/10.1007/s00294-007-0165-7){:target="_blank"}). Interestingly, 
Stc1 homologs are present in the Mucoromycota despite the lack 
of other RITS components. The HP1 complex (Swi6/Chp2 
homologs) is also conserved among the Mucoromycota, and all 
early-diverging fungi that have retained RNAi. KMT6 was lost in every Mucoromycota species, yet is conserved in some other early-diverging fungal phyla. It was also lost in some 
Ascomycota and many Basidiomycota, suggesting that **H3K27me and Polycomb silencing was lost in many independent events during fungal evolution**.

6. We searched for DNMT enzymes, specifically 
those capable of producing transcription-repressive 5mC DNA 
methylation. ...

    *TODO: add the other miscellaneous explanation mentioned in paper about Fig 1.*

7. Surprisingly, every transformation experiment aiming at clr4 deletion resulted in heterokaryotic mutants that were unable to achieve complete mutant homokaryosis, and after 20 vegetative passages they still harbored WT nuclei ([Fig S3](./images/2_S3.png){:target="_blank"}). We tested if Clr4 activity could be essential for viability by genotyping the asexual 
progeny of a heterokaryotic mutant, reasoning an equal rate of 
WT or mutant homokaryons in the spore population is expected 
for a nondeleterious mutation. To do so, 40 spores from a heterokaryotic mutant were dissected and plated on a nonselective medium and after incubation, 30 spores were able to germinate. Next, we analyzed the germinated spores and found either WT homokaryotic or heterokaryotic colonies (with WT and *clr4Δ* nuclei), yet no *clr4Δ* homokaryons were observed ([Fig S3](./images/2_S3.png){:target="_blank"}). These results indicate that **homokaryotic *clr4Δ* spores are not able to survive implying Clr4 is essential and suggesting that a functionalH3K9me-mediated heterochromatin formation mechanism is required for cell viability in *M. lusitanicus*.**



## Regions Targeted by H3K9 Methylation Are Transcriptional  Deserts Enriched with Repeats

## RNAi Is Dispensable for Heterochromatin Maintenance at TEs

## RNAi Prevents Genome Instability by Repressing GremLINE1 Transposition to Noncentromeric Regions